homeassistant/components/climate/__init__.py

```diff
@@ -325,13 +287,13 @@ def target_temperature_low(self):
         return None
 
     @property
-    def is_away_mode_on(self):
-        """Return true if away mode is on."""
+    def current_hold_mode(self):
```

Comment:
current? if current is the running state for operation, same should apply here. ie it should just be "hold_mode"